### PR TITLE
Handle multiSend for Usul proposals

### LINKED ISSUE #787: Bug: Usul | Decoded Data full breakdown
### Overview

image

currently the decodedTransactions for Usul is only showing part of the decodedTransaction data.

image

These array should be combined to show all the data.

### Steps to reproduce

1. Open a executed proposal.
2. extend executed code downdown

### Screenshots / Video

_No response_

### Version

Prod (app.fractalframework.xyz/)

### Browser

Chrome (or Chromium based)

### Operating System

Mac

### web3 wallet

MetaMask

### PR DESCRIPTION
**Pull request by mudrila**
## Description
This PR adjusts data decoding for `Usul`-based proposals and handles unpacking "internal" transactions from `multiSend`.

## Notes

N/A

## Issue / Notion doc (if applicable)

Closes #787 

## Testing

1. Create subDAO on Usul-based parent DAO
2. Verify that expanded proposal details are shown properly
3. Alternatively - there's existing DAO with such a proposal `daos/0xF332aAb56e00f7FDEdEDDC50033Ce5e2D0DD6d61/proposals/1`
4. Alternatively - just create proposal with `multiSend` execution (you'll have to decode data yourself)

## Screenshots (if applicable)
Знімок екрана 2023-01-03 о 22 32 37

### CODE REVIEW

File: src/providers/Fractal/utils/api.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ import { getChainMetadataById } from '../../Web3Data/chains';
  * @param pathname gnosis path name for request
  * @param queryParams object for requests
  * @param version gnosis api version (default: v1)
- * @url https://safe-transaction.[network].gnosis.io/
+ * @url https://safe-transaction-[network].safe.global/
  * @returns
  */
 export const buildGnosisApiUrl = (
```
Comment (Da-Colon) [inline src/providers/Fractal/utils/api.ts:12]: I we still using this function anywhere? I'm pretty all instants have been replaced with SafeService?

**Review by herbig — APPROVED** [review]
Nice one.

File: src/providers/Fractal/utils/api.ts (hunk 1 above)
Comment (Da-Colon) [reply src/providers/Fractal/utils/api.ts:12]: ah that makes sense

**Review by Da-Colon — APPROVED** [review]
LGTM

**Pull request merged**